### PR TITLE
move / copy updated

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref: #26

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
screenshot_20160731_110135
screenshot_20160731_111832

**Comment by tobiasKaminsky** [comment]
@jancborchardt please help me on the wording of the second screenshot :)

File: src/com/owncloud/android/db/PreferenceManager.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ public static void setSortAscending(Context context, boolean ascending) {
      */
     public static int getUploaderBehaviour(Context context) {
         return getDefaultSharedPreferences(context)
-                .getInt(AUTO_PREF__UPLOADER_BEHAVIOR, FileUploader.LOCAL_BEHAVIOUR_COPY);
+                .getInt(AUTO_PREF__UPLOADER_BEHAVIOR, 0);
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/db/PreferenceManager.java]: Why did you remove the constant and replaced it with a "0" (which is the same as ```FileUploader.LOCAL_BEHAVIOUR_COPY```)?

File: src/com/owncloud/android/ui/activity/FileDisplayActivity.java [hunk 2]
```diff
@@ -691,8 +695,23 @@ private void requestUploadOfFilesFromFileSystem(Intent data, int resultCode) {
                 remotePaths[j] = remotePathBase + (new File(filePaths[j])).getName();
             }
 
-            int behaviour = (resultCode == UploadFilesActivity.RESULT_OK_AND_MOVE) ? FileUploader
-                    .LOCAL_BEHAVIOUR_MOVE : FileUploader.LOCAL_BEHAVIOUR_COPY;
+            // default, as fallback
+            int behaviour = FileUploader.LOCAL_BEHAVIOUR_FORGET;
+
+            switch (resultCode) {
+                case UploadFilesActivity.RESULT_OK_AND_MOVE:
```
Comment (AndyScherzinger) [inline src/com/owncloud/android/ui/activity/FileDisplayActivity.java]: Is RESULT_OK_AND_MOVE still an option an in can be reached in code?

**Comment by AndyScherzinger** [comment]
Just some very minor things, code-wise :+1:
